src/App.jsx

```diff
@@ -1,10 +1,64 @@
 import './App.css';
+import axios from "axios";
+import {useState} from "react";
 
 function App() {
 
+    const [countries, setCountries] = useState([]);
+
+    async function fetchCountries() {
+        try {
+            // cca3 (landcode) ook meegenomen om als key te gebruiken
+            const result = await axios.get(`https://restcountries.com/v3.1/all?fields=name,flag,population,region,cca3`);
+            setCountries(result.data.sort((countryA, countryB) => countryA.population - countryB.population));
+        } catch (e) {
+            console.error(e);
+        }
+    }
+
+    function changeCountryTextColor(country) {
+        let color = '';
```

Comment:
Een variabele is hier overbodig. Je zou de string ook direct kunnen retourneren na de case.

`switch(country.region){
case 'Africa';
return 'blue'
break;
etc...
}
`